fig/rmd-06-equality.0.svg

```diff
@@ -0,0 +1,288 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
+
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   width="101.94106mm"
+   height="172.08611mm"
+   viewBox="0 0 361.20848 609.75389"
+   id="svg2"
+   version="1.1"
+   inkscape:version="0.91 r13725"
+   sodipodi:docname="rmd-06-equality.0.svg"
+   inkscape:export-filename="/home/robsyme/Projects/Education/Software Carpentry/r-novice-gapminder/fig/equality.2.png"
```

Comment:
Is there a way to generalize this export path? Otherwise I imagine it might create an error if someone else wants to edit it in the future...